spec/support/bundle_setup.rb

```diff
@@ -4,6 +4,10 @@ def bundle_setup(proj)
   PreAssembly::Bundle.new(context_from_proj(proj))
 end
 
+def bundle_setup_ar_model(proj)
+  PreAssembly::Bundle.new(context_from_proj_ar_model(proj))
+end
```

Comment:
i'm not loving the method names -- could this just be bundle_context_setup?

Reply:
#275